pkg/adafruitio/adafruitio.go

```diff
@@ -117,3 +192,12 @@ type request struct {
 type response struct {
 	Error string
 }
+
+// Feed is a struct containing feed information from AdaFruit.
```

Comment:
[nit] Feed contains metadata about an Adafruit feed.

1. Structs are obviously structs. Good comments should provide (and only provide) information that cannot be inferred from other sources.
2. `AdaFruit` should be `Adafruit`.